packages/models/src/authorization/key.ts

```diff
@@ -22,3 +22,14 @@ export const Key = z.object({
 });
 
 export type Key = z.infer<typeof Key>;
+
+export const JWKKey = z.object({
+  alg: z.string(),
+  e: z.string(),
+  kid: z.string(),
+  kty: z.string(),
+  n: z.string(),
+  use: z.string(),
+});
```

Comment:
Suggestion: what about removing the `export` from this? This requires moving `ClientJWKKey` and `ProducerJWKKey` inside this file, but in this way we make sure that `JWKKey` is not used anywhere else.

Reply:
Great suggestion, makes sense